Pipfile

```diff
@@ -20,6 +20,8 @@ requests = "~=2.26"
 geocoder = "~=1.38"
 # For date parsing and manipulation
 python-dateutil = "~=2.8"
+# For EXIF parsing
+pillow = "~=8.0"
```

Comment:
This `Pipfile` change should come together with a `Pipfile.lock` change (to make sure all the precise versions/dependencies are reproducible), but I don't see one in this diff. Maybe you forgot to add the `Pipfile.lock` change, or forgot to run `pipenv install`?